docs/docsite/rst/porting_guides/porting_guide_2.9.rst

```diff
@@ -78,6 +78,8 @@ Modules
 * If you changed any tasks to specify less restrictive permissions while using 2.9.12, those changes will be unnecessary (but will do no harm) in 2.9.13.
 * To avoid the issue raised in CVE-2020-1736, specify a ``mode`` parameter in all file-based tasks that accept it.
 
+* ``dnf`` and ``yum`` - Before version 2.9.13, the ``dnf`` module (and ``yum`` action when it uses ``dnf``)  did not validate GPG signatures of packages (CVE-2020-14365). It now does. If you see an error such as ``Failed to validate GPG signature for [package name]``, please ensure that you have imported the correct GPG key for the DNF repository and/or package you are using. One way to do this is with the ``rpm_key`` module. Although we discourage it, in some cases it may be necessary to disable the GPG check. This can be done by explicitly adding ``disable_gpg_check: yes`` in your ``dnf`` or ``yum`` task.
```

Comment:
I might reword this like:

> As of Ansible 2.9.13, the ``dnf`` module....properly validates GPG signatures...

Largely to eliminate the `It now does.` sentence.  The same with the other porting guides.